src/odemis/dataio/test/csv_test.py

```diff
@@ -240,6 +240,29 @@ def testExportSpectrumLineNoWL(self):
             raised = True
         self.assertFalse(raised, 'Failed to read csv file')
 
+    def testExportTemporalSpectrum(self):
+        """Try simple temporal spectrum export; note that size is made up and is not specific"""
+        size = (150, 340)
+        dtype = numpy.uint16
+        md = {model.MD_WL_LIST: numpy.linspace(536e-9, 650e-9, size[0]).tolist(),
```

Comment:
Actually, it should be `size[1]`, as the spectrum (C) is the second dimension. This error is also in `testExportSpectrumLine()`. Could you fix it too? (I think we should have some logging when this happens, because for now, it's silently dropped. I'll have a look at it).